src/app/collection-page/edit-collection-page/collection-source/collection-source.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="container-fluid">
     <div class="d-inline-block float-right space-children-mr">
-        <button class=" btn btn-danger" *ngIf="!(isReinstatable() | async)"
-                [disabled]="!(hasChanges() | async)"
+        <button class=" btn btn-danger" *ngIf="(isReinstatable()  | async) === false"
+                [disabled]="(hasChanges()  | async) === false"
```

Comment:
@ybnd & @artlowel  : I think @atarix83 is correct here that this may not work properly if we want to keep this rule enabled... see also this [stackoverflow answer](https://stackoverflow.com/questions/66789429/eslint-async-pipes-should-not-be-negated/67971535#67971535).   In that Stackoverflow answer, they also recommend using either `!== true` or possibly creating a custom pipe to handle this better (`| isTruthy`).   

I think we need to come to a **quick solution** to these questions or consider temporarily removing this rule and add it back in once we have a solution.  I don't want to delay the work on #2750 or upgrading to Angular 16 for much longer...so if we don't come to a solution, this work may require delaying.

(Apologies for rushing this, but we are very behind schedule for these upgrades & I'll be out-of-office on March 13-17, so I feel we must move this forward quickly)

Reply:
@tdonohue @atarix83 `!== true` makes sense
I've addressed most of the comments, will go through them once more tomorrow and push before (CET) noon